vello_shaders/src/compile/mod.rs

```diff
@@ -85,6 +85,7 @@ pub struct ShaderInfo {
 }
 
 impl ShaderInfo {
+    #[allow(clippy::result_large_err, reason = "Large error expected.")]
```

Comment:
```suggestion
    #[expect(clippy::result_large_err, reason = "Deferred: This is a cold code path.")]
```

The old `reason` just restated the lint name, whereas the reason should explain why we're ignoring the error here. I believe that our MSRV is high enough to use `expect`.

Reply:
Thank you so much! Lets see if it passes CI 😄